queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+ SELECT * FROM animals WHERE neutered = true;
+ SELECT * FROM animals WHERE name != 'Gabumon';
+ SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
you did well with your database project, kindly note that you didn't write the first query for this :

![image](https://github.com/mhnazary/vit-clinic/assets/20537712/ab4745de-d54f-4910-901f-6128e2c468c7)